kubernetes-client/pom.xml

```diff
@@ -143,6 +143,12 @@
       <scope>compile</scope>
       <optional>true</optional>
     </dependency>
+    <dependency>
+      <groupId>org.projectlombok</groupId>
+      <artifactId>lombok</artifactId>
+      <scope>compile</scope>
```

Comment:
I think the appropriate scope here should be provided here. Lombok should not be propagated transitively

Reply:
+1

Reply:
done.